modules/ossm-extensions-smextension-deploy.adoc

```diff
@@ -0,0 +1,42 @@
+////
```

Comment:
Are you going to mention this is deprecated? Or should we just remove this section?

Reply:
There's a note up in the overview about the deprecation.  We're going to leave it for now.   We'll remove it once the API is officially no longer supported.